### PR TITLE
009 autoupdate transaction info

### PR DESCRIPTION
**Pull request by dosjcq**
autoupdate transaction info

### CODE REVIEW

File: src/api/bridge.js [hunk 1]
```diff
@@ -22,29 +26,50 @@ const L2Bridge = new ethers.Contract(L2BridgeAddress, L2BridgeAbi, signer)
...
-  console.log(chainId)
+export async function bridgeToken(chainId, tokenId, setPending, setTransactionStatus, setChange, change, setDisableButtons) {
 
   const Token = Contracts[chainId].Token
   const Bridge = Contracts[chainId].Bridge
   const BridgeAddress = Contracts[chainId].BridgeAddress
-
   const signerAddress = await signer.getAddress()
   const allowance = (await Token.getApproved(tokenId)).toString()
+  console.log(chainId);
```
Comment (alex-lazarev) [inline src/api/bridge.js]: not needed

File: src/api/bridge.js [hunk 2]
```diff
@@ -22,29 +26,50 @@ const L2Bridge = new ethers.Contract(L2BridgeAddress, L2BridgeAbi, signer)
...
             Bridge.outboundTransfer(signerAddress, tokenId).then(tx => {
+              setTransactionStatus(`Outbound with hash: ${shortenAddress(tx.hash)}`);
               tx.wait().then(() => {
-                setPending(false)
+                setTransactionStatus(`Mined with hash: ${shortenAddress(tx.hash)}`);
+                setTimeout(function testTokens() {
+                  axios
+                    .get(`${host}/tokens/`)
+                    .then(response => {
+                      console.log(response.data.results[tokenId])
```
Comment (alex-lazarev) [inline src/api/bridge.js]: not needed

File: src/api/bridge.js [hunk 3]
```diff
@@ -22,29 +26,50 @@ const L2Bridge = new ethers.Contract(L2BridgeAddress, L2BridgeAbi, signer)
...
+                      if (response.data.results[tokenId].chain_id !== chainId) {
+                        setPending(false)
+                        setChange(change => !change)
+                        setDisableButtons(false)
+                        console.log('succwsss');
```
Comment (alex-lazarev) [inline src/api/bridge.js]: not needed

File: src/api/bridge.js [hunk 4]
```diff
@@ -22,29 +26,50 @@ const L2Bridge = new ethers.Contract(L2BridgeAddress, L2BridgeAbi, signer)
...
+                      } else {
+                        console.log('not succes');
```
Comment (alex-lazarev) [inline src/api/bridge.js]: not needed

File: src/api/fetchTokens.js [hunk 5]
```diff
@@ -6,6 +6,7 @@ export function getTokensInfo() {
   return axios
     .get(`${host}/tokens/`)
     .then(response => {
+      console.log(response.data.results);
```
Comment (alex-lazarev) [inline src/api/fetchTokens.js]: not needed

File: src/api/statusHandler.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+export function getTransactionStatus() {
+//   return axios
+//     .get(`${host}/tokens/`)
+//     .then(response => {
+//       return response.data.results
+//     })
+//     .catch(error => {
+//       console.log(error)
+//     })
```
Comment (alex-lazarev) [inline src/api/statusHandler.js]: instead of commenting, the code should be deleted

File: src/api/statusHandler.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+    axios
+        .get(`${host}/tokens/`)
+        .then(response => {
+        return response.data.results
+        })
+        .catch(error => {
+        console.log(error)
+    })
+}
```
Comment (alex-lazarev) [inline src/api/statusHandler.js]: a new blank line must be added

File: src/components/Gallery/Gallery.jsx [hunk 8]
```diff
@@ -48,12 +50,14 @@ function TokensList({ tokens }) {
...
 export function Gallery() {
   const [tokensList, setTokensList] = useState([])
+  const [change, setChange] = useState(true);
 
   useEffect(() => {
     getTokensInfo().then(res => {
       setTokensList(res)
     })
-  }, [])
+    console.log(change);
```
Comment (alex-lazarev) [inline src/components/Gallery/Gallery.jsx]: not needed

File: src/components/Web3Status/Web3Status.jsx [hunk 9]
```diff
@@ -39,6 +39,8 @@ function NetworkSwitcher() {
     [NetworksMenuHidden]: !menuIsVisible,
   })
 
+  console.log(supportedChainIds);
```
Comment (alex-lazarev) [inline src/components/Web3Status/Web3Status.jsx]: not needed

File: src/api/bridge.js (hunk 1 above)
Comment (thelazarev) [inline src/api/bridge.js:40]: not needed

File: src/networks.json [hunk 10]
```diff
@@ -1,38 +1,36 @@
...
-        "symbol": "ETH",
+        "name": "Ganache eth",
+        "symbol": "Eth",
         "decimals": 18
       },
-      "rpcUrls": [
-        "https://rinkeby.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161"
-      ],
-      "blockExplorerUrls": ["https://rinkeby.etherscan.io"]
+      "rpcUrls": ["https://0.0.0.0:8545"],
```
Comment (alex-lazarev) [inline src/networks.json]: https://127.0.0.1:8545

File: src/networks.json [hunk 11]
```diff
@@ -1,38 +1,36 @@
...
+      "chainName": "Ganache bsc",
       "nativeCurrency": {
-        "name": "Binance Coin",
+        "name": "Ganache bsc",
         "symbol": "BNB",
         "decimals": 18
       },
-      "rpcUrls": ["https://data-seed-prebsc-1-s1.binance.org:8545/"],
-      "blockExplorerUrls": ["https://bscscan.com"]
+      "rpcUrls": ["https://0.0.0.0:8546"],
```
Comment (alex-lazarev) [inline src/networks.json]: https://127.0.0.1:8546

**Comment by Onbridge-io** [comment]
already on main

**Pull request closed without merging**